examples/guided_onboarding_example/public/components/step_three.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useEffect, useState } from 'react';
+
+import { EuiButton, EuiSpacer, EuiText, EuiTitle, EuiTourStep } from '@elastic/eui';
+
+import { GuidedOnboardingPluginStart } from '@kbn/guided-onboarding-plugin/public/types';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiPageContentHeader_Deprecated as EuiPageContentHeader,
+  EuiPageContentBody_Deprecated as EuiPageContentBody,
+} from '@elastic/eui';
+
+interface StepThreeProps {
+  guidedOnboarding: GuidedOnboardingPluginStart;
+}
+
+export const StepThree = (props: StepThreeProps) => {
+  const {
+    guidedOnboarding: { guidedOnboardingApi },
+  } = props;
+
+  const [isTourStepOpen, setIsTourStepOpen] = useState<boolean>(false);
+
+  useEffect(() => {
+    const subscription = guidedOnboardingApi
+      ?.isGuideStepActive$('search', 'search_experience')
+      .subscribe((isStepActive) => {
+        setIsTourStepOpen(isStepActive);
+      });
+    return () => subscription?.unsubscribe();
+  }, [guidedOnboardingApi]);
+
+  return (
+    <>
+      <EuiPageContentHeader>
+        <EuiTitle>
+          <h2>
+            <FormattedMessage
+              id="guidedOnboardingExample.stepThree.title"
+              defaultMessage="Example step 3"
+            />
+          </h2>
+        </EuiTitle>
+      </EuiPageContentHeader>
+      <EuiPageContentBody>
+        <EuiText>
+          <p>
+            <FormattedMessage
+              id="guidedOnboardingExample.guidesSelection.stepThree.explanation"
+              defaultMessage="The EUI tour on this page is displayed, when a url param 'showTour' is set to 'true'."
```

Comment:
nit: I think this is the old text, because we don't use the url param anymore to display the EUItour.